.gitignore

```
@@ -1,3 +1,4 @@
 node_modules/
 package-lock.json
+simple-chat/build/
```

Вот так сделаешь и папка build будет игнорироваться в любой подпапке.

```suggestion
build
```